src/routes/notifications/notifications.service.ts

```diff
@@ -76,6 +76,27 @@ export class NotificationsService {
     }
   }
 
+  /**
+   * Sends a delete request for a chain, notification uuid and safe address.
+   * The uuid is expected to be managed by the client. Its value should be equal
+   * to the provided when the client called {@link registerDevice}.
+   *
+   * @param chainId
+   * @param uuid the same uuid provided when the device was registered.
+   * @param safeAddress
```

Comment:
I think you need to add a description for each parameter for it to show up in the documentation. It doesn't mention anything here though: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

Tooltip example (from IntelliJ)
![image](https://user-images.githubusercontent.com/3332770/218091636-749710d4-9e3b-4376-ae92-0659fe4776bd.png)

Reply:
Visual Studio Code shows the complete tooltip, but I changed it anyway in [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714) 👍 

![Screenshot 2023-02-10 at 14 44 41](https://user-images.githubusercontent.com/21182069/218108461-c92b26d9-97f6-4e3e-bc50-1fefd54afaf0.png)

Reply:
Interesting. Yeah now I see the params correctly.

![image](https://user-images.githubusercontent.com/3332770/218113150-0c4e9244-2953-4380-a4f7-0111d430ad48.png)

No big issue though! Thanks for updating!